pkg/index/job/correction/service/options_test.go

```diff
@@ -0,0 +1,360 @@
+// Copyright (C) 2019-2023 vdaas.org vald team <vald@vdaas.org>
```

Comment:
> **NOTE**
I implemented the options for the corrector object so I generated the test code for it!